### PR TITLE
Add Soundfly Cask

### PR DESCRIPTION
**Pull request by aleksandargosevski**
Please review. :)

### CODE REVIEW

**Comment by vitorgalvao** [comment]
It’s missing `uninstall`. There are [scripts](https://github.com/phinze/homebrew-cask/tree/master/developer/bin) and [documentation](https://github.com/phinze/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#uninstall-stanza-details) geared for that. Do you need any help?

File: Casks/soundfly.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'soundfly' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://abyssoft.com/software/soundfly/downloads/soundfly.zip'
+  name 'Soundfly'
+  homepage 'http://abyssoft.com/software/soundfly/'
+  license :freemium
```
Comment (vitorgalvao) [inline Casks/soundfly.rb]: Is this a free app that works indefinitely but with limitations that can be removed by paying?

**Comment by goschevski** [comment]
I found this app today, and this is my second contribution on cask, so i don't have too much experience with this. As far as I know this is totally free app.

Not sure how to add code for uninstalling app... :(

**Comment by vitorgalvao** [comment]
> As far as I know this is totally free app.

[That’s not freemium](http://en.m.wikipedia.org/wiki/Freemium). What you want is `:gratis`.

Care to take a look at the documentation and take a stab at it? We're here to help with questions.

**Comment by goschevski** [comment]
Will do.

**Comment by sammoore** [comment]
Soundfly is awesome, and so is Caskroom. I might take a stab at it, and I'm very familiar with Git and Ruby. However, do I just make a new pull request or what? I feel like there's no way to improve on this without pull requesting @goschevski's fork.

/cc @vitorgalvao

**Comment by vitorgalvao** [comment]
> do I just make a new pull request or what?

Yes. And on the top post you write “Closes #10194”, so it closes this one automatically, when merged.

**Comment by vitorgalvao** [comment]
Closing for lack of interest/implementation. If anyone is still interested in adding, please resubmit in a working condition.

**Comment by leoj3n** [comment]
Should be completed by #18583

**Pull request closed without merging**